src/tests/server.mjs

```diff
@@ -151,6 +153,127 @@ router.put("/messages/:id", (request, response) => {
   }
 })
 
+router.get("/posts", async (request, response) => {
+  const { worker_id } = request.query
+
+  const posts = await getPosts(worker_id)
+
+  const res = templateRenderer.render("./posts", { posts, worker_id })
+
+  response.type("html").status(200).send(res)
+})
+
```

Comment:
These routes are for app-like behavior (with persistence). They require a `worker_id` param to avoid flaky tests by ensuring that each test worker has its own database file. Since tests are running in parallel, a single database file would be shared between all workers.

This can still be accessed on your local browser by starting the server and visiting `http://localhost:9000/__turbo/posts?worker_id=0` (any `worker_id` is fine).